BonusCalcApi/V1/Gateways/OperativesGatewayOptions.cs

```diff
@@ -6,7 +7,7 @@ public class OperativesGatewayOptions
     {
         public const string OpGatewayOptionsName = "OperativesGatewayOptions";
         [Required]
-        public string RepairsHubBaseAddr { get; set; }
+        public Uri RepairsHubBaseUrl { get; set; }
```

Comment:
When I changed the name to match the key in appsettings.json it required me to change the type to `Uri` which in turn required all the changes elsewhere to build correctly.